tracing-serde/src/lib.rs

```diff
@@ -6,7 +6,7 @@
 //! [![Documentation (master)][docs-master-badge]][docs-master-url]
 //!
 //! [docs-badge]: https://docs.rs/tracing-serde/badge.svg
-//! [docs-url]: https://docs.rs/tracing-serde
+//! [docs-url]: crate
```

Comment:
What does `crate` mean here? I looked over the RFC (https://github.com/rust-lang/rfcs/blob/master/text/1946-intra-rustdoc-links.md) and I don't see it defined.

Reply:
It's a normal rust path: it means 'the crate this link is defined in', since crates don't know their own names. It will go to <https://docs.rs/tracing-serde> on docs.rs and to the crate root when you document locally.

Reply:
It is documented on https://doc.rust-lang.org/nightly/rustdoc/linking-to-items-by-name.html if that helps.

Reply:
Oh, thanks for the details!